### ISSUE TITLE
support injecting scripts into jsx

### ISSUE THREAD

**Issue by indifferentghost** [muratgozel/metapatcher#10]
I think this just needs to format and close script tags.

**Comment by muratgozel** [comment]
how do you inject scripts into jsx? can you provide an example?

**Comment by indifferentghost** [comment]
Sorry about the delay (note Repl.it messes with heads) 

https://replit.com/@ThomasD/metapatcher-test

If you continue to add tags it will also appear inside the script tags.

**Comment by indifferentghost** [comment]
Adding a closing script tag `</script>` has it work correctly.

**Comment by muratgozel** [comment]
hmm okay, i was about to update the library, will look into this too.

**Comment by muratgozel** [comment]
hi again @indifferentghost i have updated the library but its a major update, complete refactoring codebase with method changes, its all in the readme and the problem you mentioned should be resolved by this new version. let me know please if you decide to upgrade to version 4.

**Comment by indifferentghost** [comment]
Hey I've upgraded a couple of comments.
- Requiring `async` and `type` in `MetapatcherSetJsAttrs` seems extraneous to me.
- Using [`document.createElement` in `setStylesheet`](https://github.com/muratgozel/metapatcher/blob/master/src/index.ts#L589) requires me to shim `document` if I'm using this in the server.

**Comment by indifferentghost** [comment]
> Using [document.createElement in setStylesheet](https://github.com/muratgozel/metapatcher/blob/master/src/index.ts#L589) requires me to shim document if I'm using this in the server.

Should be said [`document.createElement` is in `setScript`](https://github.com/muratgozel/metapatcher/blob/master/src/index.ts#L541) as well.

**Comment by indifferentghost** [comment]
```ts
	metapatcher.set("link", {
		id: "pico",
		rel: "stylesheet",
		href: "https://cdn.jsdelivr.net/npm/@picocss/pico@2.0.6/css/pico.jade.min.css",
	});
	metapatcher.set("script", {
		id: "htmx",
		src: "https://unpkg.com/htmx.org@1.9.11",
		integrity: htmxIntegrety,
		crossorigin: "anonymous",
	});
```
This works wonderfully.

**Comment by indifferentghost** [comment]
All is resolved and works beautifully!
Resolved by 51ce444785bb9a556b179ff01b5dffef095ec32a

> * Requiring `async` and `type` in `MetapatcherSetJsAttrs` seems extraneous to me.
Resolved by 5c0317fd88968befce228efba23eddaa73949f3a

> * Using [`document.createElement` in `setStylesheet`](https://github.com/muratgozel/metapatcher/blob/master/src/index.ts#L589) requires me to shim `document` if I'm using this in the server.
Resolved by 55275af4c4513ce1ed9e04069fbac0d81d7c791e
